src/compressed_tensors/quantization/quant_scheme.py

```diff
@@ -47,12 +47,31 @@ class QuantizationScheme(BaseModel):
     input_activations: Optional[QuantizationArgs] = None
     output_activations: Optional[QuantizationArgs] = None
 
+    @model_validator(mode="after")
+    def validate_model_after(model: "QuantizationArgs") -> Dict[str, Any]:
+        inputs = model.input_activations
+        weights = model.weights
+        outputs = model.output_activations
+
+        if inputs is not None:
+            if inputs.actorder is not None:
+                raise ValueError("Cannot apply actorder to input activations")
+
+        if weights is not None:
+            if weights.dynamic:
+                raise ValueError("Cannot apply dynamic quantization to weights")
+
```

Comment:
*technically* this is allowed even if not a valid approach

Reply:
Sounds good. This PR is only meant to guard against a few cases not explicitly covered downstream, so feel free to suggest any other invalid combinations we want to guard against.